recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -35,7 +72,7 @@
    "metadata": {},
    "source": [
     "\n",
-    "## One-shot Prompt with Granite Code 20b\n",
+    "## One-shot Prompt with Granite Code 3b\n",
```

Comment:
I think on these we should just leave the size out wherever we can (in writing).

In the section where we call `find_langchain_model` we can either have some commented line that finds 20b with another note saying this is how to get the best quality.